tox.ini

```diff
@@ -71,7 +71,8 @@ commands =
 [testenv:typecheck]
 deps =
     attrs==21.2.0  # This version should track the version in pex/vendor/__init__.py.
-    mypy==0.800
+    mypy[python2]==0.910
```

Comment:
I mistakenly thought this upgrade was needed to support @total_ordering used in `Enum.Value` in this PR.